### PR TITLE
Update windows.md

### PR DESCRIPTION
**Pull request by flundar**
Release Notes:

- N/A

### CODE REVIEW

**Comment by flundar** [comment]
@cla-bot check

**Comment by flundar** [comment]
Last commit is legit. 

Sorry i made a mistake.

File: docs/src/development/windows.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Clone down the [Zed repository](https://github.com/zed-industries/zed).
   ```
 
 - Install [Visual Studio](https://visualstudio.microsoft.com/downloads/) with optional component `MSVC v*** - VS YYYY C++ x64/x86 build tools` and install Windows 11 or 10 SDK depending on your system
-
+- "Please ensure that at least Windows `10 SDK version 2104 (10.0.20348.0)` is installed on your machine. You can download it from the [Windows SDK Archive](https://developer.microsoft.com/windows/downloads/windows-sdk/)
```
Comment (mrnugget) [inline docs/src/development/windows.md:24]:
```suggestion
- Please ensure that at least Windows `10 SDK version 2104 (10.0.20348.0)` is installed on your machine. You can download it from the [Windows SDK Archive](https://developer.microsoft.com/windows/downloads/windows-sdk/)
```

File: docs/src/development/windows.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Clone down the [Zed repository](https://github.com/zed-industries/zed).
   ```
 
 - Install [Visual Studio](https://visualstudio.microsoft.com/downloads/) with optional component `MSVC v*** - VS YYYY C++ x64/x86 build tools` and install Windows 11 or 10 SDK depending on your system
-
+- Ensure that at least Windows `10 SDK version 2104 (10.0.20348.0)` is installed on your machine. You can download it from the [Windows SDK Archive](https://developer.microsoft.com/windows/downloads/windows-sdk/)
```
Comment (mrnugget) [inline docs/src/development/windows.md:24]: This line needs to go below the next one.

**Comment by flundar** [comment]
Waiting for approvement.

**Comment by mrnugget** [comment]
There's still a formatting error

File: docs/src/development/windows.md [hunk 3]
```diff
@@ -21,8 +21,8 @@ Clone down the [Zed repository](https://github.com/zed-industries/zed).
   ```
 
 - Install [Visual Studio](https://visualstudio.microsoft.com/downloads/) with optional component `MSVC v*** - VS YYYY C++ x64/x86 build tools` and install Windows 11 or 10 SDK depending on your system
-
```
Comment (mrnugget) [inline docs/src/development/windows.md:24]: This blank line needs to stay

Comment (flundar) [reply docs/src/development/windows.md:24]: Fixed

**Review by flundar — COMMENTED** [review]
So there is no explenation about version thing. for line 23?

**Comment by mrnugget** [comment]
> So there is no explenation about version thing. for line 23?

It's now in the parentheses in the previous line

**Pull request merged**